### PR TITLE
Upgrade Keycloak to 24.0.4

### PR DESCRIPTION
**Pull request by akunzai**
- https://github.com/keycloak/keycloak/releases/tag/24.0.4

### CODE REVIEW

**Comment by akunzai** [comment]
cc @elkman

**Comment by elkman** [comment]
Hi @akunzai,
thanks for your PR, because of the public holiday tomorrow I have a long weekend and will have a look at it next week.

I hope you are not in a hurry ;)

**Review by elkman — COMMENTED** [review]
Hi @akunzai!

thanks for the changes, looks very good.

Could you give me a hint on the background for ResteasyHttpHeaders stuff in KeycloakSessionUtils? 

The version should be changed in docs/release/1.4.14.txt as well.

**Comment by akunzai** [comment]
Hi @elkman 

See https://github.com/keycloak/keycloak/pull/27032

The resteasy-core dependency has been removed since Keycloak 24.0.0

File: keycloak-plugins/radius-plugin/src/main/java/com/github/vzakharchenko/radius/radius/handlers/session/KeycloakSessionUtils.java [hunk 1]
```diff
@@ -67,15 +62,9 @@ public static void context(KeycloakSession session,
         Resteasy.pushContext(KeycloakSession.class, session);
         Resteasy.pushContext(KeycloakTransaction.class,
                 session.getTransactionManager());
-        MultivaluedMap<String, String> headers = new MultivaluedHashMap<>();
```
[the remark this answers is not in the document]
Comment (akunzai) [reply keycloak-plugins/radius-plugin/src/main/java/com/github/vzakharchenko/radius/radius/handlers/session/KeycloakSessionUtils.java:70]: from: removal of resteasy-core (https://github.com/keycloak/keycloak/pull/27032)

**Pull request merged**